shristy-kumari/html-css-task/index.html

```diff
@@ -0,0 +1,560 @@
+                    <td>13500</td>
+                    <td><a href="#trash-popup"><i class='far fa-trash-alt'></i></a>&nbsp;&nbsp;<i class='far fa-edit'></i></td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>Bluetooth Speaker</td>
+                    <td>22/07/2024</td>
+                    <td><i class="fa fa-close"></i></td>
+                    <td>Red, Blue</td>
+                    <td>4500</td>
+                    <td>3</td>
+                    <td>13500</td>
+                    <td><i class='far fa-trash-alt'></i>&nbsp;&nbsp;<i class='far fa-edit'></i></td>
+                </tr>
+                <tr>
+                    <td>3</td>
+                    <td>Bluetooth Speaker</td>
+                    <td>22/07/2024</td>
+                    <td><i class="fas fa-check"></i></td>
+                    <td>Red, Blue</td>
+                    <td>4500</td>
+                    <td>3</td>
+                    <td>13500</td>
+                    <td><i class='far fa-trash-alt'></i>&nbsp;&nbsp;<i class='far fa-edit'></i></td>
+                </tr>
+                <tr>
+                    <td>4</td>
+                    <td>Bluetooth Speaker</td>
+                    <td>22/07/2024</td>
+                    <td><i class="fa fa-close"></i></td>
+                    <td>Red, Blue</td>
+                    <td>4500</td>
+                    <td>3</td>
+                    <td>13500</td>
+                    <td><i class='far fa-trash-alt'></i>&nbsp;&nbsp;<i class='far fa-edit'></i></td>
+                </tr>
+                <tr>
+                    <td>5</td>
+                    <td>Bluetooth Speaker</td>
+                    <td>22/07/2024</td>
+                    <td><i class="fa fa-close"></i></td>
+                    <td>Red, Blue</td>
+                    <td>4500</td>
+                    <td>3</td>
+                    <td>13500</td>
+                    <td><i class='far fa-trash-alt'></i>&nbsp;&nbsp;<i class='far fa-edit'></i></td>
+                </tr>
+                <tr>
+                    <td>6</td>
+                    <td>Bluetooth Speaker</td>
+                    <td>22/07/2024</td>
+                    <td><i class="fas fa-check"></i></td>
+                    <td>Red, Blue</td>
+                    <td>4500</td>
+                    <td>3</td>
+                    <td>13500</td>
+                    <td><i class='far fa-trash-alt'></i>&nbsp;&nbsp;<i class='far fa-edit'></i></td>
+                </tr>
+                <tr>
+                    <td>7</td>
+                    <td>Bluetooth Speaker</td>
+                    <td>22/07/2024</td>
+                    <td><i class="fas fa-check"></i></td>
+                    <td>Red, Blue</td>
+                    <td>4500</td>
+                    <td>3</td>
+                    <td>13500</td>
+                    <td><i class='far fa-trash-alt'></i>&nbsp;&nbsp;<i class='far fa-edit'></i></td>
```

Comment:
use consistent quotes for attribute values, either use double quotes or single quotes throughout the code.